pkg/manager/member/utils.go

```diff
@@ -386,3 +387,24 @@ func getTikVConfigMapForTiKVSpec(tikvSpec *v1alpha1.TiKVSpec, tc *v1alpha1.TidbC
 	}
 	return cm, nil
 }
+
+// buildStableStoreLabelsValue transform map[string]string into key1=value1,key2=value2
+// with a stable order
+func buildStableStoreLabelsValue(labels map[string]string) string {
+	if len(labels) < 1 {
+		return ""
+	}
+	s := ""
+	var keys []string
+	for k := range labels {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+	for _, key := range keys {
+		v, ok := labels[key]
+		if ok {
+			s = fmt.Sprintf("%s,%s", s, fmt.Sprintf("%s=%s", key, v))
+		}
+	}
+	return s[1:]
```

Comment:
```suggestion
    var envs []string
    for k, v := range labels {
        envs = append(envs, fmt.Sprintf("%s=%s", k, v)) 
    }   
    sort.Strings(envs)
    return strings.Join(envs, ",")
```

we should always try to use built-in functions to simplify code

btw, please add some basic units 1) nil map 2) 1 element map 3) more than 1 elements map